content/F2015/F15-event-balancinglifestyle.md

```diff
@@ -0,0 +1,23 @@
+title: Big CSters Discussion Circles: Balancing Lifestyle
+date: 2015-10-21 17:00
+category: Events
+tags: social, big csters
+slug: balancing-lifestyle
+author: Ziwei Zhao
+summary: Some of us have had times when we think one aspect of our life is more important than the other, such as studying over hanging out with friends or being with family. However, we should be aware that everything plays a role for our well-being. We will be sharing some tips and strategies to balance your super-busy student life. Come and meet your fellow CSters!
```

Comment:
This needs to be in the event body, not the summary. Please test your changes before you request a merge. I can't merge this, let alone deploy it to production. I'm redeploying this (not sure why the changes aren't propagating automatically) so you can see that this did not address my feedback.